.travis.yml

```diff
@@ -18,12 +18,19 @@ matrix:
     - npm run lint
     - npm run coverageNode
 
+  - name: chrome
+    node_js: 10
+    addons:
+      chrome: stable
```

Comment:
Funny how Travis decided to use "stable" for Chrome and "latest" (which I believe is also stable) for Firefox.

Reply:
Yep, I had to read https://docs.travis-ci.com/user/firefox to understand that latest means stable, and it looks like it's called that to match Firefox URL patterns.